From 6b2a679bbd6fa8b3053805e1e48e5108b3162990 Mon Sep 17 00:00:00 2001
From: Naushir Patuck <naush@raspberrypi.com>
Date: Wed, 29 Nov 2023 13:09:05 +0000
Subject: [PATCH 0800/1002] drivers: media: pisp_be: pisp_fe: Update UAPI
 header licenses

Update the license tags on the pisp UAPI header files with the
"Linux-syscall-note" clause.  Also replace the "GPL-2.0" tag with the
preferred "GPL-2.0-only" tag.

Signed-off-by: Naushir Patuck <naush@raspberrypi.com>
---
 drivers/media/platform/raspberrypi/pisp_be/pisp_be_config.h  | 2 +-
 drivers/media/platform/raspberrypi/rp1_cfe/pisp_common.h     | 2 +-
 drivers/media/platform/raspberrypi/rp1_cfe/pisp_fe_config.h  | 2 +-
 drivers/media/platform/raspberrypi/rp1_cfe/pisp_statistics.h | 2 +-
 drivers/media/platform/raspberrypi/rp1_cfe/pisp_types.h      | 2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/media/platform/raspberrypi/pisp_be/pisp_be_config.h b/drivers/media/platform/raspberrypi/pisp_be/pisp_be_config.h
index f8e283ef6c8c..bddb6e5d4670 100644
--- a/drivers/media/platform/raspberrypi/pisp_be/pisp_be_config.h
+++ b/drivers/media/platform/raspberrypi/pisp_be/pisp_be_config.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0-only */
+/* SPDX-License-Identifier: GPL-2.0-only WITH Linux-syscall-note */
 /*
  * PiSP Back End configuration definitions.
  *
diff --git a/drivers/media/platform/raspberrypi/rp1_cfe/pisp_common.h b/drivers/media/platform/raspberrypi/rp1_cfe/pisp_common.h
index 38db5ae4ee92..e91aa2ed3659 100644
--- a/drivers/media/platform/raspberrypi/rp1_cfe/pisp_common.h
+++ b/drivers/media/platform/raspberrypi/rp1_cfe/pisp_common.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0-only WITH Linux-syscall-note */
 /*
  * RP1 PiSP common definitions.
  *
diff --git a/drivers/media/platform/raspberrypi/rp1_cfe/pisp_fe_config.h b/drivers/media/platform/raspberrypi/rp1_cfe/pisp_fe_config.h
index c8dd2492a1db..4b76f640894b 100644
--- a/drivers/media/platform/raspberrypi/rp1_cfe/pisp_fe_config.h
+++ b/drivers/media/platform/raspberrypi/rp1_cfe/pisp_fe_config.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0-only WITH Linux-syscall-note */
 /*
  * RP1 PiSP Front End Driver Configuration structures
  *
diff --git a/drivers/media/platform/raspberrypi/rp1_cfe/pisp_statistics.h b/drivers/media/platform/raspberrypi/rp1_cfe/pisp_statistics.h
index d36e12e30bed..d8d7fcb57fec 100644
--- a/drivers/media/platform/raspberrypi/rp1_cfe/pisp_statistics.h
+++ b/drivers/media/platform/raspberrypi/rp1_cfe/pisp_statistics.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0-only WITH Linux-syscall-note */
 /*
  * RP1 PiSP Front End statistics definitions
  *
diff --git a/drivers/media/platform/raspberrypi/rp1_cfe/pisp_types.h b/drivers/media/platform/raspberrypi/rp1_cfe/pisp_types.h
index ba0d4906205d..93d2d3c27a56 100644
--- a/drivers/media/platform/raspberrypi/rp1_cfe/pisp_types.h
+++ b/drivers/media/platform/raspberrypi/rp1_cfe/pisp_types.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0-only WITH Linux-syscall-note */
 /*
  * RP1 PiSP Front End image definitions.
  *
-- 
2.44.0

